packages/mds-policy-author/tests/api.spec.ts

```diff
@@ -288,24 +291,41 @@ describe('Tests app', () => {
     })
 
     it('can publish a policy if the geo is published', async () => {
-      const policy = await createPolicyAndGeographyFactory(PolicyFactory(), { publish_date: now() })
-      const result = await request
-        .post(pathPrefix(`/policies/${policy.policy_id}/publish`))
-        .set('Authorization', POLICIES_PUBLISH_SCOPE)
-        .expect(StatusCodes.OK)
-      test.value(result).hasHeader('content-type', APP_JSON)
+      // const policy = await createPolicyAndGeographyFactory(PolicyFactory(), now())
+      let policy = null
+      try {
+        policy = await createPolicyAndGeographyFactory(PolicyFactory(), now())
+      } catch (err) {
+        // eslint-disable-next-line no-console
+        console.log(`write stuff err ${err}`)
```

Comment:
Should we not re-raise here?

Reply:
thanks! these were wrapped weirdly when i was hunting down that sub/mock problem with the kafka stream client.

Reply:
You added logging the specific error in both cases, but you're not re-raising the caught error. Is that intentional?